tools/snp.sh

```diff
@@ -594,7 +629,7 @@ build_base_qemu_cmdline() {
   add_qemu_cmdline_opts "-device virtio-net-pci,disable-legacy=on,iommu_platform=true,netdev=vmnic,romfile="
 
   # Storage
-  add_qemu_cmdline_opts "-device virtio-scsi-pci,id=scsi0,disable-legacy=on,iommu_platform=true,romfile="
+  add_qemu_cmdline_opts "-device virtio-scsi-pci,id=scsi0,disable-legacy=on,iommu_platform=true"
```

Comment:
You need to rebase off upstream code. This will override the new `romfile` requirement for the devices.

Reply:
Still need to address this